### PR TITLE
Add new sdk for ACE provisioning GSM, the sdk can be used to do partner management

### PR DESCRIPTION
**Pull request by jeffrey-ACE-zz**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Gsm/Management.Gsm/Microsoft.Azure.Management.Gsm.csproj [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			<![CDATA[
+        This is a public preview release of the Azure Gsm SDK. Included with this release is all operation to get/modify partner id for tenant
+      ]]>
+		</PackageReleaseNotes>
+	</PropertyGroup>
+	<PropertyGroup>
+		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+	</PropertyGroup>
+	<ItemGroup>
+	  <Folder Include="Properties\" />
```
Comment (shahabhijeet) [inline src/SDKs/Gsm/Management.Gsm/Microsoft.Azure.Management.Gsm.csproj]: Please remove, no need to add this directory explicitly

File: src/SDKs/Gsm/Management.Gsm/Microsoft.Azure.Management.Gsm.csproj [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.Management.Gsm</PackageId>
```
Comment (shahabhijeet) [inline src/SDKs/Gsm/Management.Gsm/Microsoft.Azure.Management.Gsm.csproj]: Fix indentation.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
What does GSM mean? What is the name of your service, why use acronym.
Please do the following in order
1) Generate your SDK using generate.cmd
2) build your project as below
msbuild build.proj /t:build /p:Scope=SDKs\Gsm

Following above steps will add meta data files.

Please also provide links to merged spec repo PRs

File: src/SDKs/Gsm/Management.Gsm/Microsoft.Azure.Management.Gsm.csproj (hunk 1 above)
Comment (jeffrey-ACE) [reply src/SDKs/Gsm/Management.Gsm/Microsoft.Azure.Management.Gsm.csproj]: @shahabhijeet  GSM stands for Global subscription management, it is also our web role name.
Thanks for reviewing the code, fix the comments. 
This is our spec repo pr: https://github.com/Azure/azure-rest-api-specs/pull/2285

**Comment by jeffrey-ACE** [comment]
@shahabhijeet after talking to my manager and PM, we decide to rename the namespace to managementPartner. Can you please continue the review?

**Comment by jeffrey-ACE** [comment]
@shahabhijeet  thank you so much for your help at Friday. My pr for swagger folder renaming is merged: https://github.com/Azure/azure-rest-api-specs/pull/2351. So regenerated the SDK and updated the PR, can you please take a look?

**Review by shahabhijeet — APPROVED** [review]

**Comment by jeffrey-ACE** [comment]
@shahabhijeet thank you so much. Can I know when will the package be published

**Pull request merged**